package.json

```diff
@@ -10,8 +10,8 @@
     "scripts": {
       "prebuild": "rm -rf ./dist",
       "build:assets": "gulp build",
-      "build:server": "tsc && cp -r src/views dist/ && cp -r src/config dist/",
-      "build": "tsc && cp -r src/views dist/",
+      "build:server": "tsc -p tsconfig-build.json && cp -r src/views dist/ && cp -r src/config dist/",
+      "build": "tsc -p tsconfig-build.json && cp -r src/views dist/",
```

Comment:
as discussed maybe `tsc --project` clearer.. I had to look up what `-p` was short for.

Reply:
Agreed, i'll change this.